tests/multigrid_evolve_test.cpp

```diff
@@ -84,14 +84,26 @@ extern QudaInverterType setup_inv[QUDA_MAX_MG_LEVEL];
 extern int num_setup_iter[QUDA_MAX_MG_LEVEL];
 extern double setup_tol[QUDA_MAX_MG_LEVEL];
 extern int setup_maxiter[QUDA_MAX_MG_LEVEL];
+extern QudaCABasis setup_ca_basis[QUDA_MAX_MG_LEVEL];
+extern int setup_ca_basis_size[QUDA_MAX_MG_LEVEL];
+extern double setup_ca_lambda_min[QUDA_MAX_MG_LEVEL];
+extern double setup_ca_lambda_max[QUDA_MAX_MG_LEVEL];
+
 extern int setup_maxiter_refresh[QUDA_MAX_MG_LEVEL];
 extern QudaSetupType setup_type;
 extern bool pre_orthonormalize;
 extern bool post_orthonormalize;
 extern double omega;
 extern QudaInverterType coarse_solver[QUDA_MAX_MG_LEVEL];
 extern QudaInverterType smoother_type[QUDA_MAX_MG_LEVEL];
+
 extern double coarse_solver_tol[QUDA_MAX_MG_LEVEL];
+extern QudaCABasis coarse_solver_ca_basis[QUDA_MAX_MG_LEVEL];
+extern int coarse_solver_ca_basis_size[QUDA_MAX_MG_LEVEL];
+extern double coarse_solver_ca_lambda_min[QUDA_MAX_MG_LEVEL];
+extern double coarse_solver_ca_lambda_max[QUDA_MAX_MG_LEVEL];
+
+
```

Comment:
clang-format suggested the following change:


```
-
```

<!-- identifier: clang-format -->
<!-- File: tests/multigrid_evolve_test.cpp -->